public/app/plugins/datasource/influxdb/components/editor/config/ConfigEditor.tsx

```diff
@@ -29,17 +29,18 @@ const versions: Array<SelectableValue<InfluxVersion>> = [
   {
     label: 'Flux',
     value: InfluxVersion.Flux,
-    description: 'Advanced data scripting and query language. Supported in InfluxDB 2.x and 1.8+',
+    description: 'Supported in InfluxDB 2.x and 1.8+',
   },
 ];
 
 const versionsWithSQL: Array<SelectableValue<InfluxVersion>> = [
-  ...versions,
+  { ...versions[0] },
```

Comment:
enum would be nice so we can give these indicies a better label

Reply:
I don't want to introduce enums for such simple code and soon (after the feature flag) we will have one list. So. I'll keep it like this :)